riff-raff/app/controllers/TargetController.scala

```diff
@@ -41,7 +41,7 @@ class TargetController(config: Config,
     val maybeTargetId = targetDynamoRepository.get(targetKey, projectName)
     maybeTargetId.map { targetId =>
       // find recent deploys of this project / stage
-      val filter = DeployFilter(projectName = Some(s"^${targetId.projectName}$$"), stage = Some(stage))
+      val filter = DeployFilter(projectName = Some(targetId.projectName), stage = Some(stage))
```

Comment:
I think the aim of this was to anchor to start and end in mongo adn thus only return exact matches. Looks like `isExactMatchProjectName` (see [here](https://github.com/guardian/riff-raff/blob/master/riff-raff/app/deployment/filter.scala#L21)) now exists to handle this case and I suggest that you need to set it to true here.

Reply:
Ah interesting! Have added this now, though it was behaving as expected without it, so perhaps this is just being more explicit?

Reply:
I think you'd find corner cases where it wouldn't such as trying to do this with `Editorial Tools::Workflow` when `Editorial Tools::Workflow::Workflow Frontend` exists.